packages/common/src/content/_fetch.ts

```diff
@@ -0,0 +1,106 @@
+import { IItem } from "@esri/arcgis-rest-portal";
```

Comment:
the exported fns in this file are not part of the public API, but are here so that they can be tested and/or mocked